models/modules/img2img_turbo/img2img_turbo.py

```diff
@@ -201,17 +201,14 @@ def forward(self, x, prompt):
         ).input_ids.cuda()
         caption_enc = self.text_encoder(caption_tokens)[0]
```

Comment:
Are you sure about the `[0]`  ?

Reply:
with refs:
1.https://huggingface.co/transformers/v4.8.0/model_doc/clip.html#flaxcliptextmodel
2.https://github.com/huggingface/transformers/blob/f91c16d270e5e3ff32fdb32ccf286d05c03dfa66/src/transformers/models/clip/modeling_clip.py#L759 
"outputs= self.text_encoder(caption_tokens)"
type(outputs)= text_encoder  <class 'transformers.modeling_outputs.BaseModelOutputWithPooling'>
len(outputs) = 2
outputs[0].shape = torch.Size([4, 77, 1024])  this is last_hidden_state
outputs[1].shape = torch.Size([4, 1024]) this is the pooler_output  
According to the explication of refs 1, should be outputs[0].